plugins/cache/bap_cache_main.ml

```diff
@@ -0,0 +1,120 @@
+open Core_kernel
```

Comment:
let's store and print everything in megabytes. 
The cdf function should still use kilobytes, but the main dimension for us should be megabytes.